src/BinSkim.Rules/PERules/BA4001.ReportPortableExecutableCompilerData.cs

```diff
@@ -5,6 +5,7 @@
 using System.Collections.Generic;
 using System.Collections.Immutable;
 using System.Composition;
+using System.IO;
```

Comment:
I think this change is a new small feature,
we can add a `NEW: ` in ReleaseHistory.md